torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8479,21 +8458,113 @@ def wrapper(t: torch.Tensor, *args, dim=None, keepdim=False, **kwargs):
 
 # Reduction operator database
 reduction_op_db: List[ReductionOpInfo] = [
+    ReductionOpInfo(
+        'amax',
+        nan_policy='propagate',
+        dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
+        sample_inputs_func=sample_inputs_amax_amin,
+        reference=reference_reduction_numpy(np.amax),
+        skips=(
+            # FIXME: dim=[] is incorrectly reducing all dimensions
+            SkipInfo('TestReductions', 'test_dim_empty'),
+            SkipInfo('TestReductions', 'test_dim_empty_keepdim'),
+        ),
+    ),
+    ReductionOpInfo(
+        'amin',
+        nan_policy='propagate',
+        dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
+        sample_inputs_func=sample_inputs_amax_amin,
+        reference=reference_reduction_numpy(np.amin),
+        skips=(
+            # FIXME: dim=[] is incorrectly reducing all dimensions
+            SkipInfo('TestReductions', 'test_dim_empty'),
+            SkipInfo('TestReductions', 'test_dim_empty_keepdim'),
+        ),
+    ),
+    ReductionOpInfo(
+        'all',
+        identity=True,
+        supports_multiple_dims=False,
+        supports_autograd=False,
+        result_dtype=torch.bool,
+        dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+        reference=reference_reduction_numpy(np.all),
+        skips=(
+            # FIXME: does not support passing keepdim without dim
+            SkipInfo('TestReductions', 'test_dim_default_keepdim'),
+            # FIXME: does not support dim=None
+            SkipInfo('TestReductions', 'test_dim_none'),
+            SkipInfo('TestReductions', 'test_dim_none_keepdim'),
+            # FIXME: uint8 input returns uint8 instead of bool
```

Comment:
There may be an issue for this

Reply:
Yup, many of these skips have issues opened already. I was lazy to add links to all of them since they're all tracked in the reduction tracking issue now.